runtime/src/lib.rs

```diff
@@ -424,7 +424,8 @@ parameter_types! {
 	pub NativeTokens: BTreeSet<(Vec<u8>, MultiLocation)> = {
 		let mut t = BTreeSet::new();
 		t.insert(("HDT".into(), MultiLocation::X2(Junction::Parent, Junction::Parachain { id: 200})));
```

Comment:
We sure this is right?

Reply:
It does not have to be there. Already removed.